notebooks/image_models/solutions/vertex_tuning_image_classification.ipynb

```diff
@@ -0,0 +1,1225 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #49.    hp_job.run()
</pre><p>maybe add <code>sync=False</code> to run asynchronously? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>